lntest/itest/log_error_whitelist.txt

```diff
@@ -284,3 +284,4 @@
 <time> [ERR] INVC: SettleHodlInvoice with preimage <hex>: invoice already canceled
 <time> [ERR] RPCS: [/invoicesrpc.Invoices/SettleInvoice]: invoice already canceled
 <time> [ERR] HSWC: ChannelLink(<chan>): outgoing htlc(<hex>) has insufficient fee: expected 33000, got 1020
+<time> [ERR] RPCS: [/lnrpc.Lightning/CloseChannel]: rpc error: code = Canceled desc = context canceled
```

Comment:
Interesting. I too saw this line popped up yesterday, something must have changed to cause it?

Reply:
Not sure...seeing it pop up more often lately. One of those things where we should stop and consider if the error white list is actually carrying out its intended goals...in this case though, this isn't a newly added error log as it's from the unary RPC interceptor. 

If I had to guess, it seems we're trying to close a channel but then the test ends successfully as we detect the channel close before the RPC client gets the final response back?

Reply:
Haha I thought about the whitelist too just yesterday, seems like it's better to ignore error loggings as long as the tests pass.

> If I had to guess, it seems we're trying to close a channel but then the test ends successfully as we detect the channel close before the RPC client gets the final response back?

Yeah I would guess so.